2022/incoming/st-nichs-reindeers-need-water.pod

```diff
@@ -0,0 +1,148 @@
+Author: oodler@cpan.org
+Title:  St. Nich's Reindeers Need H2O!
+Topic: Perl OOP, C<HASH> references, accessors
+
+=encoding utf8
+
+=head1 St. Nich's Reindeers Need H2O!
+
+Reindeer are hearty creatures, and Santa's must be the heartiest.
```

Comment:
I *think* this should be "hardy" and "hardiest", based on the context.

Reply:
I think I meant both but will side with _hearty_.